projects/plugins/jetpack/CHANGELOG.md

```diff
@@ -3,6 +3,11 @@
 ### This is a list detailing changes for all Jetpack releases.
 
 ## 13.8 - 2024-09-04
+### Enhancements
+- Launchpad modal: Apply experiment [#81287]
+
+## 13.8-beta - 2024-09-02
+>>>>>>> 798545c838 (Add changelog manually)
```

Comment:
Having git comparison slip in here, I'm also wondering why we are adding Changelog manually here? From my previous experiences this should be auto-generated once you commit your changes. Did it not get triggered?

Reply:
I didn't know about the procedure here. The CI task failed, so I just added it manually.
I'll take an additional look to update the Changelog correctly.

Reply:
Done in a proper way. There is a jetpack command: `jetpack changelogger add `.